Schema template reconciliation

miwent opened the pull request:
Completed reconciliation with Graylog template.

miwent committed:
Reviewed email, event, file entities and created hash entities information

miwent committed:
added duplicated entry for host_mac

miwent committed:
http entities review

changed http_url fields to http_uri
changed http_path to http_request_pathx

miwent committed:
policy and process entity review

Reconciled policy and process entities
updated documentation
(also updated Illuminate templates)

miwent committed:
reconciled service fields

miwent committed:
reconciled source entities

added source_id and source_type
corrected source_packets_sent analyzer from int to long

miwent committed:
reconciled sso fields

rempoved sso fields that were copies of windows fields
added two fields from the template

miwent committed:
Updated network entities

Added deprecation text to network_bytes_tx, network_bytes_rx, network_packets_tx, network_packets rx.
These fields were redundant to the source/destination fields containing the same data.

miwent committed:
template reconciliation

Reviewed entities sso, threat, trace, user

miwent committed:
completed template schema reconciliation

reviewed/updated vendor, palo, and windows entities

r00tnix reviewed (approved):
LGTM, only one was the host_derived, I could not see where we added anything with host_mac, are we removing that?

miwent commented:
> LGTM, only one was the host_derived, I could not see where we added anything with host_mac, are we removing that?

No, there's some more work to do on the schema, mainly I wanted to focus on getting the template and the schema synchronized and cleaning up any glaring issues, I've updated issue #9 to address this.

bud1979 commented:
> > LGTM, only one was the host_derived, I could not see where we added anything with host_mac, are we removing that?
> 
> No, there's some more work to do on the schema, mainly I wanted to focus on getting the template and the schema synchronized and cleaning up any glaring issues, I've updated issue #9 to address this.

Makes sense to me, feel free to merge when ready
